packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js

```diff
@@ -0,0 +1,11 @@
+export default {
```

Comment:
Can you put the existing banner as a fixture? `go1-catalog`

Reply:
The existing banner is a component name `go1-banner`. The styling was so different that I think I would better to create a new base banner

Reply:
like cm-banner?
(that would be easier to customize if we want to highlight other features)